sync_diff_inspector/diff.go

```diff
@@ -199,7 +293,10 @@ func (df *Diff) generateChunksIterator(ctx context.Context) (source.RangeIterato
 					zap.Int("id", node.GetID()),
 					zap.Reflect("chunk", node),
 					zap.String("state", node.GetState()))
-				df.cp.SetCurrentSavedID(node.GetID() + 1)
+				df.cp.SetCurrentSavedID(node.GetID())
+			}
+			if err = df.loadReport(path + "_report"); err != nil {
```

Comment:
🤔 we'd better put the report at the same level of checkpoint directory. to compare with checkpoint. report is an Independent part.

Reply:
this report should be a temp file, and will be deleted in the end of diff process

Reply:
yes, but it's strange to see a file called `checkpoint.pb_report` that has nothing to do with checkpoint.e